src/rpc/blockchain.cpp

```diff
@@ -2751,14 +2752,16 @@ static RPCHelpMan loadtxoutset()
     afile >> metadata;
```

Comment:
Since we are touching this file I think we could improved the `iostream error` that's thrown when the file size is less than 40 bytes.

```suggestion
    // Check if the file size is at least as large as the expected size of SnapshotMetadata
    if (afile.size() < sizeof(SnapshotMetadata)) {
        throw JSONRPCError(RPC_INTERNAL_ERROR, strprintf("Unable to load UTXO snapshot, "
                "file size is invalid");
    }
    afile >> metadata;

```